db/migrate/20200721231340_add_columnstarttime_to_routes.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnstarttimeToRoutes < ActiveRecord::Migration[6.0]
```

Comment:
I debated this when doing the original migration, string vs date time. I wasn't sure how the data from Megan would be formatted but I think date time is the right call

Reply:
You were definitely thinking along the right lines, but we can parse whatever we get from the FE into datetimes.  I think the datetimes will be much easier to work with in any business logic